File: 코딩 기초 트레이닝/문자열 바꿔서 찾기-p181864/문자열 바꿔서 찾기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181864

def solution(myString, pat):
    myString = list(myString)
    change = []

    for s in myString:
        if s == 'A':
            change.append('B')
        else:
            change.append('A')

    return 1 if pat in ''.join(change) else 0

File: 코딩 기초 트레이닝/문자열 바꿔서 찾기-p181864/문자열 바꿔서 찾기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181864

function solution(myString, pat) {
    myString = myString.split("").map((d) => {
        return d === 'A' ? 'B' : 'A';
    }).join('');

    return myString.includes(pat) ? 1 : 0;
}
